app/Models/User.php

```diff
@@ -732,7 +732,7 @@ public function setSmsSubscriptionTopicsAttribute($value)
      */
     public function getCausesAttribute($value)
     {
-        return ! empty($value) ? $value : [];
+        return ! empty($value) ? collect($value)->values()->all() : [];
```

Comment:
I wonder if this would make sense to break into two conditionals now for readability?

Also let's sneak in a comment explaining why we need this fix here!

```suggestion
        if (empty($value)) {
            return [];
        }
        
        // Fix formatting issue where some causes were saved as indexed objects
        // (e.g. {0: "animal_welfare", 1: "bullying"}) instead of an array.
        // Context: https://www.pivotaltracker.com/story/show/172005082
        return collect($value)->values()->all();
```